### PR TITLE
API-16161 DQ Removed all references and instances to bulk fhir

### PR DESCRIPTION
**Pull request by taylorrassmannlibertyits**

### CODE REVIEW

**Review by agarcia-va — APPROVED** [review]
LGTM! :shipit:!

File: data-query/src/test/java/gov/va/api/health/dataquery/service/controller/medicationorder/DatamartMedicationOrderTest.java [hunk 1]
```diff
@@ -19,6 +19,42 @@ public void assertReadable(String json, DatamartMedicationOrder expected) {
     assertThat(dm).isEqualTo(expected);
   }
 
+  @Test
+  public void requiredFieldsOnlySample() {
+    assertReadable("datamart-medication-order-required-fields-only.json", requiredOnlySample());
```
Comment (jblefkove-va) [inline data-query/src/test/java/gov/va/api/health/dataquery/service/controller/medicationorder/DatamartMedicationOrderTest.java:24]: What is the significance of this test?

Comment (taylorrassmannlibertyits) [reply data-query/src/test/java/gov/va/api/health/dataquery/service/controller/medicationorder/DatamartMedicationOrderTest.java:24]: The was code coverage missing for `Optional<??>` values being null. This test goes through those fields.

Comment (jblefkove-va) [reply data-query/src/test/java/gov/va/api/health/dataquery/service/controller/medicationorder/DatamartMedicationOrderTest.java:24]: In that case I'd prefer an explicit test for the lazy getters

File: data-query/src/test/java/gov/va/api/health/dataquery/service/controller/medicationorder/DatamartMedicationOrderTest.java [hunk 2]
```diff
@@ -19,6 +20,24 @@ public void assertReadable(String json, DatamartMedicationOrder expected) {
     assertThat(dm).isEqualTo(expected);
   }
 
+  @Test
+  public void lazy() {
```
Comment (jblefkove-va) [inline data-query/src/test/java/gov/va/api/health/dataquery/service/controller/medicationorder/DatamartMedicationOrderTest.java:24]: Remove public from these tests

File: data-query/src/test/java/gov/va/api/health/dataquery/service/controller/medicationorder/DatamartMedicationOrderTest.java [hunk 3]
```diff
@@ -19,6 +20,24 @@ public void assertReadable(String json, DatamartMedicationOrder expected) {
...
+    DatamartMedicationOrder dm =
+        DatamartMedicationOrder.builder()
+            .dosageInstruction(
+                DatamartMedicationOrder.DosageInstruction.builder()
+                    .asNeeded(false)
+                    .build()
+                    .asList())
+            .build();
+    assertThat(dm.dosageInstruction().get(0).dosageText()).isEqualTo(empty());
```
Comment (jblefkove-va) [inline data-query/src/test/java/gov/va/api/health/dataquery/service/controller/medicationorder/DatamartMedicationOrderTest.java:33]: This test is only checking `DosageInstruction`... let's check the lazy methods on the parent class, and other inner classes as well

**Review by jblefkove-va — APPROVED** [review]

**Pull request merged**